src/paymentservice/runtimeMetrics.js

```diff
@@ -0,0 +1,236 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
It was usually suggested to use snake_case for node.js filename, e.g.: runtime-metrics.js